search/includes/classes/class-versioning.php

```diff
@@ -360,7 +364,7 @@ public function get_previous_existing_version_number( Indexable $indexable ) {
 		$active_version_number = $this->get_active_version_number( $indexable );
 
 		// If there is no active version, we can't determine what previous is
-		if ( ! $active_version_number ) {
+		if ( is_wp_error( $active_version_number ) ) {
 			return new WP_Error( 'no-active-index-found', 'There is no active index version so the "next" version cannot be determined' );
```

Comment:
Seems to be old copy-pasta, let's fix it here.
```suggestion
			return new WP_Error( 'no-active-index-found', 'There is no active index version so the "previous" version cannot be determined' );
```